### DISCUSSION TITLE
Testing examples

### DISCUSSION THREAD

**Discussion by alexnguyennz** [refinedev/refine#6257] [Q&A] [answered]
Hi where are the testing examples on [this page](https://refine.dev/docs/further-readings/testing/#example)? Or are there any resources I can look at?

**Comment by BatuhanW** [comment, marked as answer]
Hey @alexnguyennz you can see this folder for various e2e examples: https://github.com/refinedev/refine/blob/master/cypress/e2e

**Comment by alexnguyennz** [comment]
Thank you.
